### DISCUSSION TITLE
Fast forward in a track.

### DISCUSSION THREAD

**Discussion by JUNSKIx1** [Androz2091/discord-player#1792] [Q&A] [answered]
If I'm playing a track, is there any way to fast forward? For example `queue.node.skipTo('1:22')` or smth like that?
Would be really nice if there would be a feature.

**Comment by twlite** [comment, marked as answer]
https://discord-player.js.org/docs/discord-player/class/GuildQueuePlayerNode?scrollTo=fm-seek

`queue.node.seek(duration)`. This can seek to the specific point. For example, `0` = beginning of the track and `60000` = 1 minute, etc.

**Reply by JUNSKIx1** [reply]
Thanks a lot!
